torch/distributions/binomial.py

```diff
@@ -1,5 +1,6 @@
 # mypy: allow-untyped-defs
```

Comment:
Can we remove `# mypy: allow-untyped-defs` yet?

Reply:
Currently, the PR only adds type hints on properties.

Reply:
I opened #144196 as a tracking issue for the state of annotations for distributions.